Re-enable Operations at NN-HAL

akodanka opened the pull request:
Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>

akodanka committed:
Re-enable Operations at NN-HAL

Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

akodanka committed:
Fix the validate logic in Add

The validate function for Add was incorrectly checking against
index 1 for activation function parameter.
The actual input index for activation function is 2.

Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>
